docs/2.9.0/docs/canton/usermanual/static_conf.rst

```diff
@@ -40,7 +40,7 @@ Configuration reference
 -----------------------
 
 The Canton configuration file for static properties is based on `PureConfig <https://pureconfig.github.io/>`__. PureConfig
-maps Scala case classes and their class structure into analogue configuration options (see e.g. the `PureConfig quick start <https://pureconfig.github.io/docs/#quick-start>`__ for an example). Therefore, the ultimate source of truth for all
+maps Scala case classes and their class structure into analog configuration options (see e.g. the `PureConfig quick start <https://pureconfig.github.io/docs/#quick-start>`__ for an example). Therefore, the ultimate source of truth for all
```

Comment:
I would revert "analog" -- this is definitely a quibble, but I tend to reserve "analog" to mean "not digital" and use "analogue" as the preferred spelling for something analogous. I think this is what m-w.com is telling me too, though not very clearly.